docs/user-docs/latest/plugins/installing-plugins-via-marketplace.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Installing plugins via the Marketplace - Strapi User Guide
-description: Instructions to install new plugins in a Strapi application via the Marketplace.
+title: Using the Marketplace - Strapi User Guide
+description: Instructions to install new plugins, providers and custom fields in a Strapi application via the Marketplace.
```

Comment:
I wouldn't mention custom fields here since they're only installable via plugins, and in the end they won't get their own tab